my_game.py

```diff
@@ -364,6 +343,7 @@ def setup(self):
         self.player_score = 0
 
         # Sprite lists
+        self.player_sprite = arcade.SpriteList()
```

Comment:
Why do we need a list of players? There is only one in the game.

Reply:
Deleted it